### DISCUSSION TITLE
How to use?

### DISCUSSION THREAD

**Discussion by Hybjis** [LiveSplit/LiveSplitOne#780] [Help] [answered]
I am confused about how to use this. The program runs in a web page, which means I can't see the timer or the splits when playing a game. The hotkeys also don't work when in game or not actively on the page. I tried building in case that gets me an app but couldn't get it to work.

**Comment by Ordoviz** [comment, marked as answer]
There are some workarounds for using LiveSplitOne with global hotkeys:
* [livesplitone-global-hotkeys](https://github.com/thearst3rd/livesplitone-global-hotkeys) is a Python script that reads your keypresses and sends the appropriate command to the web app over WebSocket.
* Use one of CryZe's experimental LiveSplitOne clients like [livesplit-one-desktop](https://github.com/CryZe/livesplit-one-desktop)
* Wait for an official desktop version of LiveSplitOne
